command/command_test.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestExecuteProgram(t *testing.T) {
+	var outstream []byte
+	var errstream []byte
+
+	xephyr := ExternalCommand{
+		path: "Xephyr",
```

Comment:
Please use an independent binary for testing features. That binary will be a data dependency (look at data deps in bazel) of this test, and you'd use runfiles to get the path to that binary.